cumulus/polkadot-omni-node/lib/src/lib.rs

```diff
@@ -40,14 +40,6 @@
 //! ## Examples
 //!
 //! For an example, see the [`polkadot-parachain-bin`](https://crates.io/crates/polkadot-parachain-bin) crate.
-//!
-//! ## Binary
```

Comment:
Shall we add a readme to this as well, such that in https://crates.io/crates/polkadot-omni-node-lib you see something? 

Note that I recommend using `cargo-readme` in as many places as possible, you don't have to rewrite the `.md` files. This is pretty common in Rust to make the README.md and `./src/lib.rs` basically be a mirror. 

If you end up doing this, we can also add a simple script that will run this command for a list of crates, and keeps them un sync in the CI. This will eventually tackle the age old issue of #1260. Ofc this can be a long term follow-up, not now :)

Reply:
This script would be so simple that I wouldn't add it to be honest. It should contain `cargo readme --no-license --no-title --no-indent-headings`. For now I just generated the README with the previous command, removed the same content from `polkadot-omni-node/lib/src/lib.rs` file, and added `include_str!(../README.md`). We will modify in a single place and not need to be concerned with mirroring: [922ae7f](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/922ae7f0bdde471a7bcee1b75da9c6d3e5743fcf).

Reply:
OKay, sounds good! 

All in all, I think we can then also rely on this workflow: 

1. All raw readmes are written as `README.docify.md`
2. We run `cargo build --feature=generate-readme`
3. The `lib.rs` contains `include_str!(../README.md)`